textdb/textdb-dataflow/src/main/java/edu/uci/ics/textdb/dataflow/join/Join.java

```diff
@@ -250,30 +242,22 @@ public void setInnerInputOperator(IOperator innerInputOperator) {
         this.innerOperator = innerInputOperator;
     }
     
+    public IOperator getInnerInputOperator() {
+        return this.innerOperator;
+    }
+    
     public void setOuterInputOperator(IOperator outerInputOperator) {
         this.outerOperator = outerInputOperator;
     }
+    
+    public IOperator getOuterInputOperator() {
+        return this.outerOperator;
+    }    
 
     @Override
     public Schema getOutputSchema() {
         return outputSchema;
     }
-    
-    public IOperator getOuterOperator() {
-        return outerOperator;
-    }
-
-    public void setOuterOperator(IOperator outerOperator) {
```

Comment:
`setInnerOperator` and `setOuterOperator` are replaced with `setInnerInputOperator` and `setOuterInputOperator`. This is because other operators are using `setInputOperator`